website/source/docs/commands/index.html.md

```diff
@@ -79,6 +79,23 @@ Command Options
      Joins a server to another server in the WAN pool.
 ```
 
+## Autocompletion
+
+The `consul` command features opt-in subcommand autocompletion that you can
+enable for your shell with `consul -autocomplete-install`. After doing so,
+you can invoke a new shell and use the feature.
+
+For example, assume a tab is typed at the end of each prompt line:
+
+```
+$ consul e
+event  exec
+
+$ consul r
+raft    reload  rtt
```

Comment:
`raft` is actually a subcommand of `operator` which looks like a limitation of the autocomplete library.

Reply:
Yeah I noticed that it doesn't do sub-sub commands

Reply:
I tried this out just now and it worked on subcommands too, this is pretty cool. 

```
preetha@preetha-work ~/go/src/github.com/hashicorp/consul-oss (jbs-autocomplete) $consul r
reload  rtt     
preetha@preetha-work ~/go/src/github.com/hashicorp/consul-oss (jbs-autocomplete) $consul r
reload  rtt     
preetha@preetha-work ~/go/src/github.com/hashicorp/consul-oss (jbs-autocomplete) $consul operator 
autopilot  raft       
preetha@preetha-work ~/go/src/github.com/hashicorp/consul-oss (jbs-autocomplete) $consul operator raft 
list-peers   remove-peer  
```

Reply:
I'll update the documentation and then merge this. this is ready to go